odoo/models.py

```diff
@@ -817,7 +817,7 @@ def export_data(self, fields_to_export, raw_data=False):
         fields_to_export = map(fix_import_export_id_paths, fields_to_export)
         if raw_data:
             self = self.with_context(export_raw_data=True)
-        return {'datas': self.__export_rows(fields_to_export)}
+        return {'datas': self._export_rows(fields_to_export)}
```

Comment:
For backward compatibility, I suggest to add the following line after the method definition:
```python
__export_rows = _export_rows
```